src/types/index.ts

```diff
@@ -99,3 +100,31 @@ export enum ENTITY {
     MATERIALIZATION = 'materialization',
     COLLECTION = 'collection',
 }
+
+export type SortDirection = 'asc' | 'desc';
+
+export enum TableStatuses {
+    LOADING = 'LOADING',
+    DATA_FETCHED = 'DATA_FETCHED',
+    NO_EXISTING_DATA = 'NO_EXISTING_DATA',
+    TECHNICAL_DIFFICULTIES = 'TECHNICAL_DIFFICULTIES',
+    UNMATCHED_FILTER = 'UNMATCHED_FILTER',
+}
+
+type TableStatus =
+    | TableStatuses.LOADING
+    | TableStatuses.DATA_FETCHED
+    | TableStatuses.NO_EXISTING_DATA
+    | TableStatuses.TECHNICAL_DIFFICULTIES
+    | TableStatuses.UNMATCHED_FILTER;
```

Comment:
Why are we making this a type and not using the enum directly?

Reply:
This is likely an oversight from when the original entity table components were created. I merely moved shared assets to a shared location. Will look into its removal.